### ISSUE TITLE
CPU_power

### ISSUE THREAD

**Issue by Csokis** [flightlessmango/MangoHud#688]
Hi!

CPU_power always shows 0W. Why? AMD Ryzen 7 5800X, Arch Linux and latest github version!

[https://i.imgur.com/da1Brnj.png](https://i.imgur.com/da1Brnj.png)

Sensors output:

> nct6798-isa-0290
Adapter: ISA adapter
in0:                      720.00 mV (min =  +0.00 V, max =  +1.74 V)
in1:                        1.01 V  (min =  +0.00 V, max =  +0.00 V)  ALARM
in2:                        3.39 V  (min =  +0.00 V, max =  +0.00 V)  ALARM
in3:                        3.33 V  (min =  +0.00 V, max =  +0.00 V)  ALARM
in4:                        1.01 V  (min =  +0.00 V, max =  +0.00 V)  ALARM
in5:                      904.00 mV (min =  +0.00 V, max =  +0.00 V)  ALARM
in6:                      1000.00 mV (min =  +0.00 V, max =  +0.00 V)  ALARM
in7:                        3.39 V  (min =  +0.00 V, max =  +0.00 V)  ALARM
in8:                        3.33 V  (min =  +0.00 V, max =  +0.00 V)  ALARM
in9:                      904.00 mV (min =  +0.00 V, max =  +0.00 V)  ALARM
in10:                     288.00 mV (min =  +0.00 V, max =  +0.00 V)  ALARM
in11:                     496.00 mV (min =  +0.00 V, max =  +0.00 V)  ALARM
in12:                       1.03 V  (min =  +0.00 V, max =  +0.00 V)  ALARM
in13:                     992.00 mV (min =  +0.00 V, max =  +0.00 V)  ALARM
in14:                       1.02 V  (min =  +0.00 V, max =  +0.00 V)  ALARM
fan1:                        0 RPM  (min =    0 RPM)
fan2:                      568 RPM  (min =    0 RPM)
fan3:                        0 RPM  (min =    0 RPM)
fan4:                        0 RPM  (min =    0 RPM)
fan5:                     2327 RPM  (min =    0 RPM)
fan6:                        0 RPM  (min =    0 RPM)
fan7:                      564 RPM  (min =    0 RPM)
SYSTIN:                    +31.0°C  (high = +80.0°C, hyst = +75.0°C)  sensor = thermistor
CPUTIN:                    +29.5°C  (high = +80.0°C, hyst = +75.0°C)  sensor = thermistor
AUXTIN0:                   +26.0°C    sensor = thermistor
AUXTIN1:                   +52.0°C    sensor = thermistor
AUXTIN2:                   +26.0°C    sensor = thermistor
AUXTIN3:                   +25.0°C    sensor = thermistor
PECI Agent 0 Calibration:  +30.5°C
SMBUSMASTER 1:             +60.0°C
PCH_CHIP_CPU_MAX_TEMP:      +0.0°C
PCH_CHIP_TEMP:              +0.0°C
intrusion0:               ALARM
intrusion1:               ALARM
beep_enable:              disabled

> zenpower-pci-00c3
> Adapter: PCI adapter
> SVI2_Core:     1.43 V
> SVI2_SoC:    1000.00 mV
> Tdie:         +38.0°C  (high = +95.0°C)
> Tctl:         +38.1°C
> Tccd1:        +53.2°C
> SVI2_P_Core:  21.70 W
> SVI2_P_SoC:    4.41 W
> SVI2_C_Core:  15.15 A
> SVI2_C_SoC:    4.41 A

**Comment by gort818** [comment]
https://www.phoronix.com/scan.php?page=news_item&px=Linux-5.11-Drops-k10temp-V-C

**Comment by gort818** [comment]
TLDR: `hwmon: (k10temp) Remove support for displaying voltage and current on Zen CPUs (2020-12-22 08:18:36 -0800)`

**Comment by gort818** [comment]
Oh wait you are using zenpower..

**Comment by gort818** [comment]
Ok yes can confim this issue.

found the offending commit here https://github.com/flightlessmango/MangoHud/commit/37e18279e78d36e6634f4c5644c7766524b94118

Should be sorted out soon

**Comment by gort818** [comment]
should be fixed in latest master

**Comment by Csokis** [comment]
Thank you! :hugs: It works perfectly! :+1:

**Comment by gort818** [comment]
Fixed in fb975309650d7b75b8f69875f5aa6fd5001bbedb
